client/src/redux/reducers/allocations.reducers.js

```diff
@@ -68,18 +74,34 @@ export function allocations(state = initialState, action) {
           ...action.payload.loading
         }
       };
-    case 'SET_SEARCH_TERM':
+    case 'MANAGE_USERS_INIT':
+      return {
+        ...state,
+        loadingUsernames: {
+          ...state.loadingUsernames,
+          ...action.payload.loading
```

Comment:
It seems like we could lose a layer of depth here, by just passing the payload rather than nesting it in a `loading` object